packages/tasks/src/library-ui-elements.ts

```diff
@@ -72,7 +72,7 @@ model = BaseModel.from_pretrained("${model.id}")`,
 ];
 
 function get_base_diffusers_model(model: ModelData): string {
-	return model.cardData?.base_model ?? "fill-in-base-model";
+	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
```

Comment:
I guess it's ok to assume we only have 1 base model for diffusion models

Reply:
yes, i hesitated with taking the 1st element but 🤷

honestly we should have specified / enforced an array of string from the beginning, it's always kinda messy to do it after the fact